packages/libs/client/src/utils/tests/parseAsPactValue.test.ts

```diff
@@ -43,9 +43,11 @@ describe('parseType', () => {
     expect(parseAsPactValue(() => 'test')).toEqual('test');
```

Comment:
Not part of this PR, but is executing those functions a potential security concern?

Reply:
WDYM? As a potential injection in the Pact code?

That should be covered by proper xss protection on the hosting website, I guess?